sofy_sofy/README.md

```diff
@@ -0,0 +1,52 @@
+# SOFY
+
+## Overview
+
+SOFY is a no-code platform for creating automated tests on mobile Apps. Users can integrate with their CI/CD pipelines to execute tests on real devices and view the results of their functional tests along with performance metrics.
+
+With this integration, you have deeper insight into your testing process by visualizing key metrics and trends such as load time, network, memory utilization and CPU. The out-of-the-box dashboard provides real-time visibility into your SOFY test results, enabling you to monitor and analyze performance over time, and make data-driven decisions to improve overall software quality.
+
+## Data Collected
+### Metrics
+
+See [metadata.csv][8] for the full list of metrics provided by this check.
+
+
+## Setup
+Get set up in just a few steps:
+
+1. Go to your [Datadog Integrations page][1] and click on the SOFY tile. Then go to the **Configuration** tab and click **Install Integration** at the bottom.
+
+1. Click **Connect Accounts** to begin authorization of the SOFY integration from the Login page in SOFY.
+
+3. Log in to [SOFY][2]: Start by logging in to SOFY and navigating to the Account Setting page by selecting Account -> Account Setting from the left-hand navigation menu.
+
+4. Connect Datadog to your SOFY Account: Once on the Account Setting page, select the [Integration tab][3] and locate the Datadog tile. Click the "Connect" button to begin the integration process.
```

Comment:
Can we just navigate from the Datadog integration tile directly to the Sofy Integration tab step 4? This way the user will automatically be on the right page after logging into the `onboarding_url`. Additionally, you will be able to use the `site` parameter to determine which datacenter the user was redirected from (e.g. `app.datadoghq.com`, `us5.datadoghq.com`, `app.datadoghq.eu`, etc). See [docs](https://docs.datadoghq.com/developers/authorization/oauth2_in_datadog/).

```suggestion
2. Click **Connect Accounts** to redirect to the [Integration tab][3] under Account Settings in SOFY.

3. Log in with your SOFY account and connect Datadog to SOFY: Start by logging in to SOFY, and then click the "Connect" button on the Datadog tile to begin the integration process.
```

Note: you will likely have to update your OAuth client's `onboarding_url` to be `https://portal.sofy.ai/app/user-settings?selectedTab=integration` and republish your app in the developer platform.